openwisp-qa-check

```diff
@@ -117,11 +136,22 @@ runcheckmigrations() {
 }
 
 runflake8() {
-  flake8 && echo "SUCCESS: Flake8 check successful!" ||
-    {
-      echoerr "ERROR: Flake8 check failed!"
-      FAILURE=1
-    }
+  if [ -n "$IGNORE_PATTERN" ]; then
```

Comment:
Do we really need this `if-else`, if the `ignore-pattern` flag is not provided, `IGNORE_PATTERN` would be an empty string, which should work